purchases/src/main/kotlin/com/revenuecat/purchases/Purchases.kt

```diff
@@ -1235,8 +1236,7 @@ class Purchases @VisibleForTesting(otherwise = VisibleForTesting.PRIVATE) intern
             version = null
         )
 
-        @get:VisibleForTesting(otherwise = VisibleForTesting.NONE)
-        @set:VisibleForTesting(otherwise = VisibleForTesting.NONE)
+        @JvmSynthetic
```

Comment:
Originally, these annotations were added to remove the property from Java code, since `internal` properties are visible from Java. `@JvmSynthetic` is a better approach.